README.md

```diff
@@ -58,9 +58,15 @@ If you're curious about the edge cases that `gosu` handles, see [`Dockerfile.tes
 
 ## Alternatives
 
-### `su-exec`
+### `ncopa/su-exec`
 
-As mentioned in `INSTALL.md`, [`su-exec`](https://github.com/ncopa/su-exec) is a very minimal re-write of `gosu` in C, making for a much smaller binary, and is available in the `main` Alpine package repository.
+As mentioned in `INSTALL.md`, [`ncopa/su-exec`](https://github.com/ncopa/su-exec) is a very minimal re-write of `gosu` in C, making for a much smaller binary, and is available in the `main` Alpine package repository.
+
+The ncopa's su-exec have critical bugs (or features), it will use root permission silently if invalid user-spec given !
```

Comment:
This line leads me to believe there's been a grave misunderstanding of the intended purpose / use case for `gosu` and `su-exec`.

The entire point of this project is for a Docker image to be able to start as `root`, and then use `gosu` to become non-root (to "step down", as it were).  The way it does this is by using Docker's _own_ `user:group` parsing code for Docker's `--user` flag (which comes from `runc` these days), and this is where the "empty string is `root`" edge case comes from.

Reply:
I'm agree "empty string is `root`", but "invalid string is `root`" is not acceptable. It will swallow typo or error configuration.

Reply:
Ah, great idea; let's add some false negatives to our test suite.  FWIW, `gosu` has sane behavior here:

```console
$ ./gosu-amd64 0day true
error: failed switching to "0day": unable to find user 0day: no matching entries in passwd file
$ echo $?
1
```

Also, this is probably a simple fix to `su-exec` too (although it has been a bit since I looked at the code over there).

Reply:
https://github.com/tianon/gosu/pull/60 :+1:

Reply:
https://github.com/tianon/gosu/commit/fd60171def742cc35f31e2076bc24c4b94b422c9
https://github.com/tianon/gosu/commit/f87df69c868e19f7258b4facb7c2472d76d98dda
https://github.com/ncopa/su-exec/pull/26

:+1: